### DISCUSSION TITLE
Plans for multiple Scene support?

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#14129] [General]
Over the last few releases I've seen the code change to start using UIScene and UISceneDelegate, so I am assuming at some point there is a plan to support multiple scenes.

Is there? and if so, any timescale you can share?

**Comment by OrlaM** [comment]
Yes we plan to support multiple scenes, but no concrete timelines at the moment. It will be at least a few months but should be some time this year.

**Reply by gutley** [reply]
Thanks for the confirmation
